tests/checks/core/test_latency.py

```diff
@@ -0,0 +1,40 @@
+import time
+
+from aioresponses import CallbackResult
+
+from checks.core.latency import run
+
+
+async def test_positive(mock_aioresponses):
+    def callback(url, **kwargs):
+        time.sleep(0.01)
+        return CallbackResult(status=200)
+
+    url = "http://server.local/__heartbeat__"
+    mock_aioresponses.get(url, callback=callback)
+
+    status, data = await run(url, max_milliseconds=20)
+
+    assert status is True
+    assert 10 <= data < 20
+
+
+async def test_negative(mock_aioresponses):
+    def callback(url, **kwargs):
+        time.sleep(0.01)
+        return CallbackResult(status=200)
+
+    url = "http://server.local/__heartbeat__"
+    mock_aioresponses.get(url, callback=callback)
+
+    status, data = await run(url, max_milliseconds=9)
```

Comment:
I just learned that since Python 3.5, `time.sleep()` is guaranteed to sleep at least for the time given, so this should always return `False`.